Makefile.am

```diff
@@ -8,7 +8,12 @@ if BUILD_OCTAVE_MOD
 octave_dir = Octave
 endif
 
-SUBDIRS = M4 man doc Win32 src $(octave_dir) examples regtest tests programs
+SUBDIRS = M4 Win32 src examples tests
+
+if FULL_SUITE
+SUBDIRS += man doc $(octave_dir) regtest programs
+endif
+
 DIST_SUBDIRS = M4 man doc Win32 src Octave examples regtest tests programs
```

Comment:
I can understand that would be a problem for the `tests` directory, but the `examples` directory contains no tests.